internal/core/app_status_report.go

```diff
@@ -345,6 +344,9 @@ func (op *statusReportOperation) Do(
 		return nil, status.Errorf(codes.FailedPrecondition, "unsupported status operation type")
 	}
 
+	// Add the time generated to the outer status report
+	realMsg.TimeGenerated = report.TimeGenerated
```

Comment:
This feels like something the UI will likely want to include (cc @jgwhite ), so I added it to the top level StatusReport. It also means we don't need to dig into the lower-level status report in the CLI at all anymore.

Reply:
Yes, this is going to be super helpful in the UI too.